README.md

```diff
@@ -425,14 +425,13 @@ package com.example.client;
 import java.util.HashMap;
 
 import com.clearblade.java.api.*;
-import com.clearblade.java.api.internal.*;
 
 public class MQTTClientJava {
 
-	private static Message message;
+	private static MQTTClient mqttClient;
 	private static boolean isInit = false;
 	
-	public static void main(String[] args) {
+	public static void Main(String[] args) {
```

Comment:
I thought main was always lowercase, are things changed now? or is it something else?